astronomer/providers/amazon/aws/example_dags/example_emr_job_flow_manual_steps.py

```diff
@@ -0,0 +1,104 @@
+"""Example DAG's for AWS EMR related operator and sensor"""
+
+import os
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.models.baseoperator import chain
+from airflow.providers.amazon.aws.operators.emr import (
+    EmrAddStepsOperator,
+    EmrCreateJobFlowOperator,
+    EmrTerminateJobFlowOperator,
+)
+
+from astronomer.providers.amazon.aws.sensors.emr import EmrStepSensorAsync
+
+JOB_FLOW_ROLE = os.environ.get("EMR_JOB_FLOW_ROLE", "EMR_EC2_DefaultRole")
+SERVICE_ROLE = os.environ.get("EMR_SERVICE_ROLE", "EMR_DefaultRole")
+AWS_CONN_ID = os.environ.get("ASTRO_AWS_CONN_ID", "aws_default")
+EMR_CONN_ID = os.environ.get("ASTRO_EMR_CONN_ID", "emr_default")
+
+
+SPARK_STEPS = [
+    {
+        "Name": "calculate_pi",
+        "ActionOnFailure": "CONTINUE",
+        "HadoopJarStep": {
+            "Jar": "command-runner.jar",
+            "Args": ["/usr/lib/spark/bin/run-example", "SparkPi", "10"],
+        },
+    }
+]
+
+JOB_FLOW_OVERRIDES = {
+    "Name": "PiCalc",
+    "ReleaseLabel": "emr-5.29.0",
+    "Applications": [{"Name": "Spark"}],
+    "Instances": {
+        "InstanceGroups": [
+            {
+                "Name": "Primary node",
+                "Market": "ON_DEMAND",
+                "InstanceRole": "MASTER",
+                "InstanceType": "m4.large",
+                "InstanceCount": 1,
+            },
+        ],
+        "KeepJobFlowAliveWhenNoSteps": False,
+        "TerminationProtected": False,
+    },
+    "JobFlowRole": JOB_FLOW_ROLE,
+    "ServiceRole": SERVICE_ROLE,
+}
+
+default_args = {
+    "execution_timeout": timedelta(minutes=30),
+    "trigger_rule": "all_done",
+}
+
+with DAG(
+    dag_id="example_emr_job_flow_manual_steps",
+    schedule_interval=None,
+    start_date=datetime(2022, 1, 1),
+    default_args=default_args,
+    tags=["example", "async", "emr"],
```

Comment:
Same suggestion as above to have tags as constants in order to avoid duplicates, typos, etc. 
E.g We might have inconsistencies where write `examples` vs `example`, or `EMR` vs `emr`. 
In general, good if we can have static strings values as constants.